database/buildSchema/44_data_display.sql

```diff
@@ -29,3 +29,12 @@ CREATE TABLE data_display_column_definition (
     UNIQUE (table_name, column_name)
 );
 
+-- Table for cataloguing all "data" tables, including lookup tables
+CREATE TABLE data_table (
+    id serial PRIMARY KEY,
+    table_name varchar NOT NULL UNIQUE,
+    name varchar,
```

Comment:
Just to clarify -- "table_name" is the actual Postgres table name (without the "data_table_" prefix), whereas "name" is the non-converted name.

E.g. if we enter "List of Ingredients" as the "name", the "table_name" will be "list_of_ingredient"

Reply:
Should we rename `name` to `description` or `display_name`. I think it would be clearer

Reply:
Yeah, I think "display_name" seems appropriate. I'll update it (will need to change front-end too).

Good suggestion, I wasn't particularly happy with it.

Reply:
Here: 49a3ca01

Front-end updated too.